src/main/java/frc/robot/Calibrations.java

```diff
@@ -13,4 +13,8 @@ public static final class AutoBalance {
 
     public static final double SET_X_TIME_LEFT_SECONDS = PLACEHOLDER_DOUBLE;
   }
+
+  public static final class AutoScore {
+    public static final double AUTO_INTAKE_TIMEOUT_SEC = PLACEHOLDER_DOUBLE;
```

Comment:
tbh let's just move this into the class itself. I don't think it's going to be re-used.

Or maybe instead of a timeout we can add a second event where we cancel the intaking like in the old 2.5 where we did `eventMap.put("retractIntake", new InstantCommand(() -> {}, lift, intake));`

Reply:
I have been informed by Nick that the minibot is likely to fall over when driving with the arm up, so it should move back with the arm down in the intaking position and just stop spinning the motors.

I added an event called "stopIntaking" which does grabber.stopMotors() on the path back

Reply:
I also changed intakeSequence to not move back to STARTING position and just end by stopping the grabber's motors